build_projects/dotnet-cli-build/CompileTargets.cs

```diff
@@ -1,402 +1,402 @@
+                foreach (var project in PackageTargets.ProjectsToPack) 
+                { 
+                    // Just build them, we'll pack later 
+                    var packBuildResult = DotNetCli.Stage1.Build( 
+                        "--build-base-path", 
+                        packagingOutputDir, 
+                        "--configuration", 
+                        configuration, 
+                        Path.Combine(c.BuildContext.BuildDirectory, "src", project)) 
+                        .Execute(); 
+ 
+                    packBuildResult.EnsureSuccessful(); 
+                } 
+            } 
+ 
+            CleanOutputDir(Path.Combine(Dirs.Stage2, "sdk")); 
+            FS.CopyRecursive(Dirs.Stage2, Dirs.Stage2Symbols); 
+ 
+            RemovePdbsFromDir(Path.Combine(Dirs.Stage2, "sdk")); 
+ 
+            return c.Success(); 
+        } 
+ 
+        private static void CleanOutputDir(string directory) 
+        { 
+            foreach (var file in FilesToClean) 
+            { 
+                FS.RmFilesInDirRecursive(directory, file); 
+            } 
+        } 
+ 
+        private static void RemovePdbsFromDir(string directory) 
+        { 
+            FS.RmFilesInDirRecursive(directory, "*.pdb"); 
+        } 
+ 
+        private static BuildTargetResult CompileCliSdk( 
+            BuildTargetContext c, 
+            DotNetCli dotnet, 
+            string rootOutputDirectory, 
+            bool generateNugetPackagesArchive = false) 
+        { 
+            var configuration = c.BuildContext.Get<string>("Configuration"); 
+            var buildVersion = c.BuildContext.Get<BuildVersion>("BuildVersion"); 
+            var srcDir = Path.Combine(c.BuildContext.BuildDirectory, "src"); 
+            var sdkOutputDirectory = Path.Combine(rootOutputDirectory, "sdk", buildVersion.NuGetVersion); 
+ 
+            CopySharedFramework(Dirs.SharedFrameworkPublish, rootOutputDirectory); 
+ 
+            FS.CleanBinObj(c, srcDir); 
+            Rmdir(sdkOutputDirectory); 
+            Mkdirp(sdkOutputDirectory); 
+ 
+            foreach (var project in ProjectsToPublish) 
+            { 
+                dotnet.Publish( 
+                    "--native-subdirectory", 
+                    "--output", sdkOutputDirectory, 
+                    "--configuration", configuration, 
+                    "--version-suffix", buildVersion.CommitCountString, 
+                    Path.Combine(srcDir, project)) 
+                    .Execute() 
+                    .EnsureSuccessful(); 
+            } 
+ 
+            FixModeFlags(sdkOutputDirectory); 
+ 
+            string compilersProject = Path.Combine(Dirs.RepoRoot, "src", "compilers"); 
+            dotnet.Publish(compilersProject, 
+                    "--output", 
+                    sdkOutputDirectory, 
+                    "--framework", 
+                    "netcoreapp1.0") 
+                    .Execute() 
+                    .EnsureSuccessful(); 
```

Comment:
I accidently deleted the file and then reverted. I see it is showing up as a new file. Need to figure out how to get git to do what I want here.